rest-server/src/models/job.js

```diff
@@ -95,6 +95,7 @@ class Job {
               createdTime: frameworkInfo.firstRequestTimestamp || new Date(2018, 1, 1).getTime(),
               completedTime: frameworkInfo.frameworkCompletedTimestamp,
               appExitCode: frameworkInfo.applicationExitCode,
+              virtualCluster: frameworkInfo.queue,
```

Comment:
Please also add 'virtualCluster' field to the result of job details API (GET /jobs/<jobName>).

Reply:
Sure.